## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 514828`
Commit: `9632df5681e5a9472271e18bd47c50d7ab62cfa3`

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 10 packages marked as broken and skipped:</summary>
  <ul>
    <li>picosnitch</li>
    <li>picosnitch.dist</li>
    <li>python313Packages.multiscale-spatial-image</li>
    <li>python313Packages.multiscale-spatial-image.dist</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python314Packages.multiscale-spatial-image</li>
    <li>python314Packages.multiscale-spatial-image.dist</li>
    <li>python314Packages.reflex-chakra</li>
    <li>python314Packages.reflex-chakra.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 13 packages failed to build:</summary>
  <ul>
    <li>python313Packages.bambi</li>
    <li>python313Packages.bambi.dist</li>
    <li>python313Packages.cartopy</li>
    <li>python313Packages.cartopy.dist</li>
    <li>python313Packages.swh-scheduler</li>
    <li>python313Packages.swh-scheduler.dist</li>
    <li>python314Packages.cartopy</li>
    <li>python314Packages.cartopy.dist</li>
    <li>python314Packages.opensfm</li>
    <li>python314Packages.opensfm.dist</li>
    <li>python314Packages.swh-scheduler</li>
    <li>python314Packages.swh-scheduler.dist</li>
    <li>swh</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 766 packages built:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>alpaca</li>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>canaille</li>
    <li>canaille.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>contact</li>
    <li>contact.dist</li>
    <li>crewai (python313Packages.crewai)</li>
    <li>crewai.dist (python313Packages.crewai.dist)</li>
    <li>cve-bin-tool</li>
    <li>cve-bin-tool.dist</li>
    <li>das</li>
    <li>das.dist</li>
    <li>deeptools</li>
    <li>deeptools.dist</li>
    <li>flatten-references-graph</li>
    <li>flatten-references-graph.dist</li>
    <li>heretic (python313Packages.heretic-llm)</li>
    <li>heretic.dist (python313Packages.heretic-llm.dist)</li>
    <li>hp2p</li>
    <li>jadx</li>
    <li>libretranslate (python313Packages.libretranslate)</li>
    <li>libretranslate.dist (python313Packages.libretranslate.dist)</li>
    <li>mapproxy</li>
    <li>mapproxy.dist</li>
    <li>marimo (python313Packages.marimo)</li>
    <li>marimo.dist (python313Packages.marimo.dist)</li>
    <li>meshtastic</li>
    <li>meshtastic.dist</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>multiqc</li>
    <li>multiqc.dist</li>
    <li>napari</li>
    <li>napari.dist</li>
    <li>newelle</li>
    <li>ome_zarr (python313Packages.ome-zarr)</li>
    <li>ome_zarr.dist (python313Packages.ome-zarr.dist)</li>
    <li>oterm</li>
    <li>oterm.dist</li>
    <li>overturemaps</li>
    <li>overturemaps.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>py3dtiles (python313Packages.py3dtiles)</li>
    <li>py3dtiles.dist (python313Packages.py3dtiles.dist)</li>
    <li>pyspread</li>
    <li>pyspread.dist</li>
    <li>python313Packages.aiosql</li>
    <li>python313Packages.aiosql.dist</li>
    <li>python313Packages.aiosql.doc</li>
    <li>python313Packages.altair</li>
    <li>python313Packages.altair.dist</li>
    <li>python313Packages.amd-quark</li>
    <li>python313Packages.amd-quark.dist</li>
    <li>python313Packages.argos-translate-files</li>
    <li>python313Packages.argos-translate-files.dist</li>
    <li>python313Packages.argostranslate</li>
    <li>python313Packages.argostranslate.dist</li>
    <li>python313Packages.arviz</li>
    <li>python313Packages.arviz-base</li>
    <li>python313Packages.arviz-base.dist</li>
    <li>python313Packages.arviz-plots</li>
    <li>python313Packages.arviz-plots.dist</li>
    <li>python313Packages.arviz.dist</li>
    <li>python313Packages.async-geotiff</li>
    <li>python313Packages.async-geotiff.dist</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.ax-platform.dist</li>
    <li>python313Packages.backtesting</li>
    <li>python313Packages.backtesting.dist</li>
    <li>python313Packages.basemap</li>
    <li>python313Packages.basemap.dist</li>
    <li>python313Packages.bokeh</li>
    <li>python313Packages.bokeh.dist</li>
    <li>python313Packages.bqplot</li>
    <li>python313Packages.bqplot.dist</li>
    <li>python313Packages.bqscales</li>
    <li>python313Packages.bqscales.dist</li>
    <li>python313Packages.bsuite</li>
    <li>python313Packages.bsuite.dist</li>
    <li>python313Packages.bumps</li>
    <li>python313Packages.bumps.dist</li>
    <li>python313Packages.cf-xarray</li>
    <li>python313Packages.cf-xarray.dist</li>
    <li>python313Packages.cltk</li>
    <li>python313Packages.cltk.dist</li>
    <li>python313Packages.cmdstanpy</li>
    <li>python313Packages.cmdstanpy.dist</li>
    <li>python313Packages.coiled</li>
    <li>python313Packages.coiled.dist</li>
    <li>python313Packages.compressai</li>
    <li>python313Packages.compressai.dist</li>
    <li>python313Packages.corner</li>
    <li>python313Packages.corner.dist</li>
    <li>python313Packages.dash</li>
    <li>python313Packages.dash-bootstrap-components</li>
    <li>python313Packages.dash-bootstrap-components.dist</li>
    <li>python313Packages.dash-bootstrap-templates</li>
    <li>python313Packages.dash-bootstrap-templates.dist</li>
    <li>python313Packages.dash.dist</li>
    <li>python313Packages.datashader</li>
    <li>python313Packages.datashader.dist</li>
    <li>python313Packages.energyflow</li>
    <li>python313Packages.energyflow.dist</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.experiment-utilities.dist</li>
    <li>python313Packages.explorerscript</li>
    <li>python313Packages.explorerscript.dist</li>
    <li>python313Packages.extra-streamlit-components</li>
    <li>python313Packages.extra-streamlit-components.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.folium</li>
    <li>python313Packages.folium.dist</li>
    <li>python313Packages.formulaic</li>
    <li>python313Packages.formulaic.dist</li>
    <li>python313Packages.geoarrow-pyarrow</li>
    <li>python313Packages.geoarrow-pyarrow.dist</li>
    <li>python313Packages.geodatasets</li>
    <li>python313Packages.geodatasets.dist</li>
    <li>python313Packages.geopandas</li>
    <li>python313Packages.geopandas.dist</li>
    <li>python313Packages.geoparquet</li>
    <li>python313Packages.geoparquet.dist</li>
    <li>python313Packages.gradio</li>
    <li>python313Packages.gradio.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.holoviews</li>
    <li>python313Packages.holoviews.dist</li>
    <li>python313Packages.hvplot</li>
    <li>python313Packages.hvplot.dist</li>
    <li>python313Packages.igraph</li>
    <li>python313Packages.igraph.dist</li>
    <li>python313Packages.inequality</li>
    <li>python313Packages.inequality.dist</li>
    <li>python313Packages.intake</li>
    <li>python313Packages.intake.dist</li>
    <li>python313Packages.iplotx</li>
    <li>python313Packages.iplotx.dist</li>
    <li>python313Packages.ipydatagrid</li>
    <li>python313Packages.ipydatagrid.dist</li>
    <li>python313Packages.ipydatawidgets</li>
    <li>python313Packages.ipydatawidgets.dist</li>
    <li>python313Packages.itables</li>
    <li>python313Packages.itables.dist</li>
    <li>python313Packages.kmapper</li>
    <li>python313Packages.kmapper.dist</li>
    <li>python313Packages.kokoro</li>
    <li>python313Packages.kokoro.dist</li>
    <li>python313Packages.labelbox</li>
    <li>python313Packages.labelbox.dist</li>
    <li>python313Packages.layoutparser</li>
    <li>python313Packages.layoutparser.dist</li>
    <li>python313Packages.leidenalg</li>
    <li>python313Packages.leidenalg.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.libpysal</li>
    <li>python313Packages.libpysal.dist</li>
    <li>python313Packages.lida</li>
    <li>python313Packages.lida.dist</li>
    <li>python313Packages.lifelines</li>
    <li>python313Packages.lifelines.dist</li>
    <li>python313Packages.linearmodels</li>
    <li>python313Packages.linearmodels.dist</li>
    <li>python313Packages.livelossplot</li>
    <li>python313Packages.livelossplot.dist</li>
    <li>python313Packages.llama-cloud</li>
    <li>python313Packages.llama-cloud.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-google-genai</li>
    <li>python313Packages.llama-index-embeddings-google-genai.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-node-parser-docling</li>
    <li>python313Packages.llama-index-node-parser-docling.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-milvus</li>
    <li>python313Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.mapclassify</li>
    <li>python313Packages.mapclassify.dist</li>
    <li>python313Packages.meshtastic</li>
    <li>python313Packages.meshtastic.dist</li>
    <li>python313Packages.mirakuru</li>
    <li>python313Packages.mirakuru.dist</li>
    <li>python313Packages.mlcroissant</li>
    <li>python313Packages.mlcroissant.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mmengine</li>
    <li>python313Packages.mmengine.dist</li>
    <li>python313Packages.momepy</li>
    <li>python313Packages.momepy.dist</li>
    <li>python313Packages.morecantile</li>
    <li>python313Packages.morecantile.dist</li>
    <li>python313Packages.msticpy</li>
    <li>python313Packages.msticpy.dist</li>
    <li>python313Packages.napari</li>
    <li>python313Packages.napari.dist</li>
    <li>python313Packages.narwhals</li>
    <li>python313Packages.narwhals.dist</li>
    <li>python313Packages.nbsmoke</li>
    <li>python313Packages.nbsmoke.dist</li>
    <li>python313Packages.neurokit2</li>
    <li>python313Packages.neurokit2.dist</li>
    <li>python313Packages.niaarm</li>
    <li>python313Packages.niaarm.dist</li>
    <li>python313Packages.niworkflows</li>
    <li>python313Packages.niworkflows.dist</li>
    <li>python313Packages.nutpie</li>
    <li>python313Packages.nutpie.dist</li>
    <li>python313Packages.odc-geo</li>
    <li>python313Packages.odc-geo.dist</li>
    <li>python313Packages.odc-loader</li>
    <li>python313Packages.odc-loader.dist</li>
    <li>python313Packages.odc-stac</li>
    <li>python313Packages.odc-stac.dist</li>
    <li>python313Packages.ome-zarr-models</li>
    <li>python313Packages.ome-zarr-models.dist</li>
    <li>python313Packages.opensfm</li>
    <li>python313Packages.opensfm.dist</li>
    <li>python313Packages.optuna</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.optuna-dashboard.dist</li>
    <li>python313Packages.optuna.dist</li>
    <li>python313Packages.osmnx</li>
    <li>python313Packages.osmnx.dist</li>
    <li>python313Packages.osmpythontools</li>
    <li>python313Packages.osmpythontools.dist</li>
    <li>python313Packages.pandantic</li>
    <li>python313Packages.pandantic.dist</li>
    <li>python313Packages.pandas-stubs</li>
    <li>python313Packages.pandas-stubs.dist</li>
    <li>python313Packages.pandera</li>
    <li>python313Packages.pandera.dist</li>
    <li>python313Packages.panel</li>
    <li>python313Packages.panel.dist</li>
    <li>python313Packages.pdfplumber</li>
    <li>python313Packages.pdfplumber.dist</li>
    <li>python313Packages.pint-xarray</li>
    <li>python313Packages.pint-xarray.dist</li>
    <li>python313Packages.plopp</li>
    <li>python313Packages.plopp.dist</li>
    <li>python313Packages.plotly</li>
    <li>python313Packages.plotly.dist</li>
    <li>python313Packages.plotnine</li>
    <li>python313Packages.plotnine.dist</li>
    <li>python313Packages.prophet</li>
    <li>python313Packages.prophet.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pyannote-pipeline</li>
    <li>python313Packages.pyannote-pipeline.dist</li>
    <li>python313Packages.pybids</li>
    <li>python313Packages.pybids.dist</li>
    <li>python313Packages.pydantic-zarr</li>
    <li>python313Packages.pydantic-zarr.dist</li>
    <li>python313Packages.pygmt</li>
    <li>python313Packages.pygmt.dist</li>
    <li>python313Packages.pyinterp</li>
    <li>python313Packages.pyinterp.dist</li>
    <li>python313Packages.pymc</li>
    <li>python313Packages.pymc.dist</li>
    <li>python313Packages.pymoo</li>
    <li>python313Packages.pymoo.dist</li>
    <li>python313Packages.pyproj</li>
    <li>python313Packages.pyproj.dist</li>
    <li>python313Packages.pyreadstat</li>
    <li>python313Packages.pyreadstat.dist</li>
    <li>python313Packages.pytest-postgresql</li>
    <li>python313Packages.pytest-postgresql.dist</li>
    <li>python313Packages.pythreejs</li>
    <li>python313Packages.pythreejs.dist</li>
    <li>python313Packages.qcodes</li>
    <li>python313Packages.qcodes-contrib-drivers</li>
    <li>python313Packages.qcodes-contrib-drivers.dist</li>
    <li>python313Packages.qcodes.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex.dist</li>
    <li>python313Packages.rio-tiler</li>
    <li>python313Packages.rio-tiler.dist</li>
    <li>python313Packages.rioxarray</li>
    <li>python313Packages.rioxarray.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.sasmodels</li>
    <li>python313Packages.sasmodels.dist</li>
    <li>python313Packages.scipp</li>
    <li>python313Packages.scipp.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.skops</li>
    <li>python313Packages.skops.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.skytemple-dtef</li>
    <li>python313Packages.skytemple-dtef.dist</li>
    <li>python313Packages.skytemple-files</li>
    <li>python313Packages.skytemple-files.dist</li>
    <li>python313Packages.skytemple-ssb-debugger</li>
    <li>python313Packages.skytemple-ssb-debugger.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.spatial-image</li>
    <li>python313Packages.spatial-image.dist</li>
    <li>spyder (python313Packages.spyder)</li>
    <li>python313Packages.spyder-kernels</li>
    <li>python313Packages.spyder-kernels.dist</li>
    <li>spyder.dist (python313Packages.spyder.dist)</li>
    <li>python313Packages.sqlframe</li>
    <li>python313Packages.sqlframe.dist</li>
    <li>python313Packages.st-pages</li>
    <li>python313Packages.st-pages.dist</li>
    <li>streamlit (python313Packages.streamlit)</li>
    <li>python313Packages.streamlit-avatar</li>
    <li>python313Packages.streamlit-avatar.dist</li>
    <li>python313Packages.streamlit-card</li>
    <li>python313Packages.streamlit-card.dist</li>
    <li>python313Packages.streamlit-echarts</li>
    <li>python313Packages.streamlit-echarts.dist</li>
    <li>python313Packages.streamlit-folium</li>
    <li>python313Packages.streamlit-folium.dist</li>
    <li>python313Packages.streamlit-kpi-card</li>
    <li>python313Packages.streamlit-kpi-card.dist</li>
    <li>python313Packages.streamlit-notify</li>
    <li>python313Packages.streamlit-notify.dist</li>
    <li>streamlit.dist (python313Packages.streamlit.dist)</li>
    <li>python313Packages.swh-auth</li>
    <li>python313Packages.swh-auth.dist</li>
    <li>python313Packages.swh-core</li>
    <li>python313Packages.swh-core.dist</li>
    <li>python313Packages.swh-export</li>
    <li>python313Packages.swh-export.dist</li>
    <li>python313Packages.swh-journal</li>
    <li>python313Packages.swh-journal.dist</li>
    <li>python313Packages.swh-objstorage</li>
    <li>python313Packages.swh-objstorage.dist</li>
    <li>python313Packages.swh-scanner</li>
    <li>python313Packages.swh-scanner.dist</li>
    <li>python313Packages.swh-storage</li>
    <li>python313Packages.swh-storage.dist</li>
    <li>python313Packages.swh-web-client</li>
    <li>python313Packages.swh-web-client.dist</li>
    <li>python313Packages.synergy</li>
    <li>python313Packages.synergy.dist</li>
    <li>python313Packages.tcia-utils</li>
    <li>python313Packages.tcia-utils.dist</li>
    <li>python313Packages.templateflow</li>
    <li>python313Packages.templateflow.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.textnets</li>
    <li>python313Packages.textnets.dist</li>
    <li>python313Packages.traittypes</li>
    <li>python313Packages.traittypes.dist</li>
    <li>python313Packages.translatehtml</li>
    <li>python313Packages.translatehtml.dist</li>
    <li>python313Packages.txtai</li>
    <li>python313Packages.txtai.dist</li>
    <li>python313Packages.vega</li>
    <li>python313Packages.vega.dist</li>
    <li>python313Packages.vegafusion</li>
    <li>python313Packages.vegafusion.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>python313Packages.wktutils</li>
    <li>python313Packages.wktutils.dist</li>
    <li>python313Packages.xarray</li>
    <li>python313Packages.xarray-dataclass</li>
    <li>python313Packages.xarray-dataclass.dist</li>
    <li>python313Packages.xarray-dataclasses</li>
    <li>python313Packages.xarray-dataclasses.dist</li>
    <li>python313Packages.xarray-einstats</li>
    <li>python313Packages.xarray-einstats.dist</li>
    <li>python313Packages.xarray.dist</li>
    <li>python313Packages.xclim</li>
    <li>python313Packages.xclim.dist</li>
    <li>python314Packages.aiosql</li>
    <li>python314Packages.aiosql.dist</li>
    <li>python314Packages.aiosql.doc</li>
    <li>python314Packages.altair</li>
    <li>python314Packages.altair.dist</li>
    <li>python314Packages.amd-quark</li>
    <li>python314Packages.amd-quark.dist</li>
    <li>python314Packages.arviz-base</li>
    <li>python314Packages.arviz-base.dist</li>
    <li>python314Packages.arviz-plots</li>
    <li>python314Packages.arviz-plots.dist</li>
    <li>python314Packages.async-geotiff</li>
    <li>python314Packages.async-geotiff.dist</li>
    <li>python314Packages.ax-platform</li>
    <li>python314Packages.ax-platform.dist</li>
    <li>python314Packages.backtesting</li>
    <li>python314Packages.backtesting.dist</li>
    <li>python314Packages.basemap</li>
    <li>python314Packages.basemap.dist</li>
    <li>python314Packages.bokeh</li>
    <li>python314Packages.bokeh.dist</li>
    <li>python314Packages.bqplot</li>
    <li>python314Packages.bqplot.dist</li>
    <li>python314Packages.bqscales</li>
    <li>python314Packages.bqscales.dist</li>
    <li>python314Packages.bsuite</li>
    <li>python314Packages.bsuite.dist</li>
    <li>python314Packages.bumps</li>
    <li>python314Packages.bumps.dist</li>
    <li>python314Packages.cf-xarray</li>
    <li>python314Packages.cf-xarray.dist</li>
    <li>python314Packages.cmdstanpy</li>
    <li>python314Packages.cmdstanpy.dist</li>
    <li>python314Packages.coiled</li>
    <li>python314Packages.coiled.dist</li>
    <li>python314Packages.compressai</li>
    <li>python314Packages.compressai.dist</li>
    <li>python314Packages.crewai</li>
    <li>python314Packages.crewai.dist</li>
    <li>python314Packages.dash</li>
    <li>python314Packages.dash-bootstrap-components</li>
    <li>python314Packages.dash-bootstrap-components.dist</li>
    <li>python314Packages.dash-bootstrap-templates</li>
    <li>python314Packages.dash-bootstrap-templates.dist</li>
    <li>python314Packages.dash.dist</li>
    <li>python314Packages.datashader</li>
    <li>python314Packages.datashader.dist</li>
    <li>python314Packages.experiment-utilities</li>
    <li>python314Packages.experiment-utilities.dist</li>
    <li>python314Packages.explorerscript</li>
    <li>python314Packages.explorerscript.dist</li>
    <li>python314Packages.extra-streamlit-components</li>
    <li>python314Packages.extra-streamlit-components.dist</li>
    <li>python314Packages.folium</li>
    <li>python314Packages.folium.dist</li>
    <li>python314Packages.formulaic</li>
    <li>python314Packages.formulaic.dist</li>
    <li>python314Packages.geoarrow-pyarrow</li>
    <li>python314Packages.geoarrow-pyarrow.dist</li>
    <li>python314Packages.geodatasets</li>
    <li>python314Packages.geodatasets.dist</li>
    <li>python314Packages.geopandas</li>
    <li>python314Packages.geopandas.dist</li>
    <li>python314Packages.geoparquet</li>
    <li>python314Packages.geoparquet.dist</li>
    <li>python314Packages.gradio</li>
    <li>python314Packages.gradio.dist</li>
    <li>python314Packages.great-tables</li>
    <li>python314Packages.great-tables.dist</li>
    <li>python314Packages.heretic-llm</li>
    <li>python314Packages.heretic-llm.dist</li>
    <li>python314Packages.holoviews</li>
    <li>python314Packages.holoviews.dist</li>
    <li>python314Packages.hvplot</li>
    <li>python314Packages.hvplot.dist</li>
    <li>python314Packages.igraph</li>
    <li>python314Packages.igraph.dist</li>
    <li>python314Packages.inequality</li>
    <li>python314Packages.inequality.dist</li>
    <li>python314Packages.intake</li>
    <li>python314Packages.intake.dist</li>
    <li>python314Packages.iplotx</li>
    <li>python314Packages.iplotx.dist</li>
    <li>python314Packages.ipydatagrid</li>
    <li>python314Packages.ipydatagrid.dist</li>
    <li>python314Packages.ipydatawidgets</li>
    <li>python314Packages.ipydatawidgets.dist</li>
    <li>python314Packages.itables</li>
    <li>python314Packages.itables.dist</li>
    <li>python314Packages.kmapper</li>
    <li>python314Packages.kmapper.dist</li>
    <li>python314Packages.labelbox</li>
    <li>python314Packages.labelbox.dist</li>
    <li>python314Packages.layoutparser</li>
    <li>python314Packages.layoutparser.dist</li>
    <li>python314Packages.leidenalg</li>
    <li>python314Packages.leidenalg.dist</li>
    <li>python314Packages.libpysal</li>
    <li>python314Packages.libpysal.dist</li>
    <li>python314Packages.lida</li>
    <li>python314Packages.lida.dist</li>
    <li>python314Packages.lifelines</li>
    <li>python314Packages.lifelines.dist</li>
    <li>python314Packages.linearmodels</li>
    <li>python314Packages.linearmodels.dist</li>
    <li>python314Packages.livelossplot</li>
    <li>python314Packages.livelossplot.dist</li>
    <li>python314Packages.mapclassify</li>
    <li>python314Packages.mapclassify.dist</li>
    <li>python314Packages.marimo</li>
    <li>python314Packages.marimo.dist</li>
    <li>python314Packages.meshtastic</li>
    <li>python314Packages.meshtastic.dist</li>
    <li>python314Packages.mirakuru</li>
    <li>python314Packages.mirakuru.dist</li>
    <li>python314Packages.mlcroissant</li>
    <li>python314Packages.mlcroissant.dist</li>
    <li>python314Packages.mlflow</li>
    <li>python314Packages.mlflow.dist</li>
    <li>python314Packages.mmcv</li>
    <li>python314Packages.mmcv.dist</li>
    <li>python314Packages.mmengine</li>
    <li>python314Packages.mmengine.dist</li>
    <li>python314Packages.momepy</li>
    <li>python314Packages.momepy.dist</li>
    <li>python314Packages.morecantile</li>
    <li>python314Packages.morecantile.dist</li>
    <li>python314Packages.msticpy</li>
    <li>python314Packages.msticpy.dist</li>
    <li>python314Packages.napari</li>
    <li>python314Packages.napari.dist</li>
    <li>python314Packages.narwhals</li>
    <li>python314Packages.narwhals.dist</li>
    <li>python314Packages.nbsmoke</li>
    <li>python314Packages.nbsmoke.dist</li>
    <li>python314Packages.neurokit2</li>
    <li>python314Packages.neurokit2.dist</li>
    <li>python314Packages.niaarm</li>
    <li>python314Packages.niaarm.dist</li>
    <li>python314Packages.niworkflows</li>
    <li>python314Packages.niworkflows.dist</li>
    <li>python314Packages.odc-geo</li>
    <li>python314Packages.odc-geo.dist</li>
    <li>python314Packages.odc-loader</li>
    <li>python314Packages.odc-loader.dist</li>
    <li>python314Packages.odc-stac</li>
    <li>python314Packages.odc-stac.dist</li>
    <li>python314Packages.ome-zarr-models</li>
    <li>python314Packages.ome-zarr-models.dist</li>
    <li>python314Packages.optuna</li>
    <li>python314Packages.optuna-dashboard</li>
    <li>python314Packages.optuna-dashboard.dist</li>
    <li>python314Packages.optuna.dist</li>
    <li>python314Packages.osmnx</li>
    <li>python314Packages.osmnx.dist</li>
    <li>python314Packages.osmpythontools</li>
    <li>python314Packages.osmpythontools.dist</li>
    <li>python314Packages.pandantic</li>
    <li>python314Packages.pandantic.dist</li>
    <li>python314Packages.pandas-stubs</li>
    <li>python314Packages.pandas-stubs.dist</li>
    <li>python314Packages.pandera</li>
    <li>python314Packages.pandera.dist</li>
    <li>python314Packages.panel</li>
    <li>python314Packages.panel.dist</li>
    <li>python314Packages.pdfplumber</li>
    <li>python314Packages.pdfplumber.dist</li>
    <li>python314Packages.pint-xarray</li>
    <li>python314Packages.pint-xarray.dist</li>
    <li>python314Packages.plopp</li>
    <li>python314Packages.plopp.dist</li>
    <li>python314Packages.plotly</li>
    <li>python314Packages.plotly.dist</li>
    <li>python314Packages.plotnine</li>
    <li>python314Packages.plotnine.dist</li>
    <li>python314Packages.prophet</li>
    <li>python314Packages.prophet.dist</li>
    <li>python314Packages.py3dtiles</li>
    <li>python314Packages.py3dtiles.dist</li>
    <li>python314Packages.pyannote-audio</li>
    <li>python314Packages.pyannote-audio.dist</li>
    <li>python314Packages.pyannote-pipeline</li>
    <li>python314Packages.pyannote-pipeline.dist</li>
    <li>python314Packages.pybids</li>
    <li>python314Packages.pybids.dist</li>
    <li>python314Packages.pydantic-zarr</li>
    <li>python314Packages.pydantic-zarr.dist</li>
    <li>python314Packages.pygmt</li>
    <li>python314Packages.pygmt.dist</li>
    <li>python314Packages.pyinterp</li>
    <li>python314Packages.pyinterp.dist</li>
    <li>python314Packages.pymoo</li>
    <li>python314Packages.pymoo.dist</li>
    <li>python314Packages.pyproj</li>
    <li>python314Packages.pyproj.dist</li>
    <li>python314Packages.pyreadstat</li>
    <li>python314Packages.pyreadstat.dist</li>
    <li>python314Packages.pytest-postgresql</li>
    <li>python314Packages.pytest-postgresql.dist</li>
    <li>python314Packages.pythreejs</li>
    <li>python314Packages.pythreejs.dist</li>
    <li>python314Packages.qcodes</li>
    <li>python314Packages.qcodes-contrib-drivers</li>
    <li>python314Packages.qcodes-contrib-drivers.dist</li>
    <li>python314Packages.qcodes.dist</li>
    <li>python314Packages.reflex</li>
    <li>python314Packages.reflex.dist</li>
    <li>python314Packages.rio-tiler</li>
    <li>python314Packages.rio-tiler.dist</li>
    <li>python314Packages.rioxarray</li>
    <li>python314Packages.rioxarray.dist</li>
    <li>python314Packages.sagemaker-mlflow</li>
    <li>python314Packages.sagemaker-mlflow.dist</li>
    <li>python314Packages.sasmodels</li>
    <li>python314Packages.sasmodels.dist</li>
    <li>python314Packages.scipp</li>
    <li>python314Packages.scipp.dist</li>
    <li>python314Packages.shiny</li>
    <li>python314Packages.shiny.dist</li>
    <li>python314Packages.skops</li>
    <li>python314Packages.skops.dist</li>
    <li>python314Packages.skytemple-dtef</li>
    <li>python314Packages.skytemple-dtef.dist</li>
    <li>python314Packages.skytemple-files</li>
    <li>python314Packages.skytemple-files.dist</li>
    <li>python314Packages.skytemple-ssb-debugger</li>
    <li>python314Packages.skytemple-ssb-debugger.dist</li>
    <li>python314Packages.smolagents</li>
    <li>python314Packages.smolagents.dist</li>
    <li>python314Packages.spacy-loggers</li>
    <li>python314Packages.spacy-loggers.dist</li>
    <li>python314Packages.spatial-image</li>
    <li>python314Packages.spatial-image.dist</li>
    <li>python314Packages.spyder</li>
    <li>python314Packages.spyder-kernels</li>
    <li>python314Packages.spyder-kernels.dist</li>
    <li>python314Packages.spyder.dist</li>
    <li>python314Packages.sqlframe</li>
    <li>python314Packages.sqlframe.dist</li>
    <li>python314Packages.st-pages</li>
    <li>python314Packages.st-pages.dist</li>
    <li>python314Packages.streamlit</li>
    <li>python314Packages.streamlit-avatar</li>
    <li>python314Packages.streamlit-avatar.dist</li>
    <li>python314Packages.streamlit-card</li>
    <li>python314Packages.streamlit-card.dist</li>
    <li>python314Packages.streamlit-echarts</li>
    <li>python314Packages.streamlit-echarts.dist</li>
    <li>python314Packages.streamlit-folium</li>
    <li>python314Packages.streamlit-folium.dist</li>
    <li>python314Packages.streamlit-kpi-card</li>
    <li>python314Packages.streamlit-kpi-card.dist</li>
    <li>python314Packages.streamlit-notify</li>
    <li>python314Packages.streamlit-notify.dist</li>
    <li>python314Packages.streamlit.dist</li>
    <li>python314Packages.swh-auth</li>
    <li>python314Packages.swh-auth.dist</li>
    <li>python314Packages.swh-core</li>
    <li>python314Packages.swh-core.dist</li>
    <li>python314Packages.swh-export</li>
    <li>python314Packages.swh-export.dist</li>
    <li>python314Packages.swh-journal</li>
    <li>python314Packages.swh-journal.dist</li>
    <li>python314Packages.swh-objstorage</li>
    <li>python314Packages.swh-objstorage.dist</li>
    <li>python314Packages.swh-scanner</li>
    <li>python314Packages.swh-scanner.dist</li>
    <li>python314Packages.swh-storage</li>
    <li>python314Packages.swh-storage.dist</li>
    <li>python314Packages.swh-web-client</li>
    <li>python314Packages.swh-web-client.dist</li>
    <li>python314Packages.synergy</li>
    <li>python314Packages.synergy.dist</li>
    <li>python314Packages.tcia-utils</li>
    <li>python314Packages.tcia-utils.dist</li>
    <li>python314Packages.templateflow</li>
    <li>python314Packages.templateflow.dist</li>
    <li>python314Packages.traittypes</li>
    <li>python314Packages.traittypes.dist</li>
    <li>python314Packages.txtai</li>
    <li>python314Packages.txtai.dist</li>
    <li>python314Packages.vega</li>
    <li>python314Packages.vega.dist</li>
    <li>python314Packages.vegafusion</li>
    <li>python314Packages.vegafusion.dist</li>
    <li>python314Packages.wandb</li>
    <li>python314Packages.wandb.dist</li>
    <li>python314Packages.whisperx</li>
    <li>python314Packages.whisperx.dist</li>
    <li>python314Packages.wktutils</li>
    <li>python314Packages.wktutils.dist</li>
    <li>python314Packages.xarray</li>
    <li>python314Packages.xarray-dataclass</li>
    <li>python314Packages.xarray-dataclass.dist</li>
    <li>python314Packages.xarray-dataclasses</li>
    <li>python314Packages.xarray-dataclasses.dist</li>
    <li>python314Packages.xarray-einstats</li>
    <li>python314Packages.xarray-einstats.dist</li>
    <li>python314Packages.xarray.dist</li>
    <li>python314Packages.xclim</li>
    <li>python314Packages.xclim.dist</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>rembg</li>
    <li>rembg.dist</li>
    <li>sasview</li>
    <li>sasview.dist</li>
    <li>skytemple</li>
    <li>skytemple.dist</li>
    <li>sptk</li>
    <li>strictdoc</li>
    <li>strictdoc.dist</li>
    <li>supercell-wx</li>
    <li>theharvester</li>
    <li>theharvester.dist</li>
    <li>tocpdf</li>
    <li>tocpdf.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>python313Packages.bambi</summary>
<pre>adding &#x27;bambi/terms/utils.py&#x27;
adding &#x27;bambi-0.17.2.dist-info/licenses/LICENSE&#x27;
adding &#x27;bambi-0.17.2.dist-info/METADATA&#x27;
adding &#x27;bambi-0.17.2.dist-info/WHEEL&#x27;
adding &#x27;bambi-0.17.2.dist-info/top_level.txt&#x27;
adding &#x27;bambi-0.17.2.dist-info/RECORD&#x27;
removing build/bdist.linux-aarch64/wheel
Successfully built bambi-0.17.2-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/bambi-0.17.2...OK
Repacking wheel as ./bambi-0.17.2-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;pythonRuntimeDepsCheckHook&quot; }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for bambi-0.17.2-py3-none-any.whl
  - arviz-plots&lt;1,&gt;=0.8.0 not satisfied by version 1.1.0
  - pymc&lt;6,&gt;=5.27.0 not satisfied by version 6.0.0</pre>
</details>
<details>
<summary>python313Packages.cartopy</summary>
<pre>FAILED tests/mpl/test_axes.py::test_geoaxes_no_subslice - Failed: Error: Image files did not match.
FAILED tests/mpl/test_axes.py::test_geoaxes_set_boundary_clipping - Failed: Error: Image files did not match.
FAILED tests/mpl/test_contour.py::test_plot_after_contour_doesnt_shrink[contour] - AssertionError: 
FAILED tests/mpl/test_gridliner.py::test_gridliner_labels_zoom - assert [&lt;cartopy.mpl...5a44ec0&gt;, ...] == [&lt;cartopy.mpl...5a44ec0&gt;, ...]
FAILED tests/mpl/test_images.py::test_imshow - Failed: Error: Image files did not match.
FAILED tests/mpl/test_images.py::test_stock_img - Failed: Error: Image files did not match.
FAILED tests/mpl/test_images.py::test_pil_Image - Failed: Error: Image files did not match.
FAILED tests/mpl/test_images.py::test_background_img - Failed: Error: Image files did not match.
FAILED tests/mpl/test_mpl_integration.py::test_cursor_values - AssertionError: assert b&#x27;-181.5, 50 ... 178.500000E)&#x27; == b&#x27;-181.5, 50 ... 1...
FAILED tests/mpl/test_pseudo_color.py::test_pcolormesh_datalim - AssertionError: 
FAILED tests/mpl/test_set_extent.py::test_extents - AssertionError: 
FAILED tests/mpl/test_set_extent.py::test_get_extent - AssertionError: 
FAILED tests/mpl/test_ticker.py::test_LatitudeFormatter_mercator - AssertionError: assert [&#x27;79.8166°S&#x27;,....7746°N&#x27;, ...] == [&#x27;80°S&#x27;, &#x27;60°..., ...
FAILED tests/test_crs_transform_vectors.py::TestTransformVectors::test_invalid_y_domain - AssertionError: 
FAILED tests/test_crs_transform_vectors.py::TestTransformVectors::test_invalid_xy_domain_corner - AssertionError: 
FAILED tests/test_polygon.py::TestMisc::test_infinite_loop_bounds[InterruptedGoodeHomolosine] - assert 3148724.315906706 == nan ± ???
FAILED tests/test_polygon.py::TestMisc::test_infinite_loop_bounds[Mollweide] - assert 0.0 == nan ± ???
FAILED tests/test_polygon.py::TestMisc::test_tiny_point_between_boundary_points - AssertionError: Got area 85832.84574015415, expecting ~81336
FAILED tests/test_vector_transform.py::Test_vector_scalar_to_grid::test_with_transform - AssertionError: 
= 39 failed, 625 passed, 7 skipped, 219 deselected, 2 xfailed, 7 warnings in 18.37s =</pre>
</details>
<details>
<summary>python313Packages.swh-scheduler</summary>
<pre>  +     &#x27;kwargs&#x27;: &#x27;[Filtered]&#x27;,
  -     &#x27;args&#x27;: [
  -         &#x27;arg&#x27;,
  -     ],
  -     &#x27;kwargs&#x27;: {
  -         &#x27;kwarg&#x27;: &#x27;kwarg&#x27;,
  -     },
        &#x27;task_name&#x27;: &#x27;swh.scheduler.tests.tasks.error&#x27;,
    }
====== 2 failed, 660 passed, 1 xfailed, 349 warnings in 163.31s (0:02:43) ======
stopping postgresql
waiting for server to shut down....2026-05-17 16:01:04.653 UTC [523] LOG:  received fast shutdown request
2026-05-17 16:01:04.653 UTC [523] LOG:  aborting any active transactions
2026-05-17 16:01:04.656 UTC [523] LOG:  background worker &quot;logical replication launcher&quot; (PID 529) exited with exit code 1
2026-05-17 16:01:04.657 UTC [524] LOG:  shutting down
2026-05-17 16:01:04.657 UTC [524] LOG:  checkpoint starting: shutdown immediate
2026-05-17 16:01:04.672 UTC [524] LOG:  checkpoint complete: wrote 975 buffers (6.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.014 s, sync=0.001 s, total=0.016 s; sync files=317, longest=0.001 s, average=0.000 s; distance=4507 kB, estimate=4507 kB; lsn=0/1953678, redo lsn=0/1953678
2026-05-17 16:01:04.687 UTC [523] LOG:  database system is shut down
 done
server stopped</pre>
</details>
<details>
<summary>python314Packages.cartopy</summary>
<pre>FAILED tests/mpl/test_axes.py::test_geoaxes_no_subslice - Failed: Error: Image files did not match.
FAILED tests/mpl/test_axes.py::test_geoaxes_set_boundary_clipping - Failed: Error: Image files did not match.
FAILED tests/mpl/test_contour.py::test_plot_after_contour_doesnt_shrink[contour] - AssertionError: 
FAILED tests/mpl/test_gridliner.py::test_gridliner_labels_zoom - assert [&lt;cartopy.mpl...1094050&gt;, ...] == [&lt;cartopy.mpl...1094050&gt;, ...]
FAILED tests/mpl/test_images.py::test_imshow - Failed: Error: Image files did not match.
FAILED tests/mpl/test_images.py::test_stock_img - Failed: Error: Image files did not match.
FAILED tests/mpl/test_images.py::test_pil_Image - Failed: Error: Image files did not match.
FAILED tests/mpl/test_images.py::test_background_img - Failed: Error: Image files did not match.
FAILED tests/mpl/test_mpl_integration.py::test_cursor_values - AssertionError: assert b&#x27;-181.5, 50 ... 178.500000E)&#x27; == b&#x27;-181.5, 50 ... 1...
FAILED tests/mpl/test_pseudo_color.py::test_pcolormesh_datalim - AssertionError: 
FAILED tests/mpl/test_set_extent.py::test_extents - AssertionError: 
FAILED tests/mpl/test_set_extent.py::test_get_extent - AssertionError: 
FAILED tests/mpl/test_ticker.py::test_LatitudeFormatter_mercator - AssertionError: assert [&#x27;79.8166°S&#x27;,....7746°N&#x27;, ...] == [&#x27;80°S&#x27;, &#x27;60°..., ...
FAILED tests/test_crs_transform_vectors.py::TestTransformVectors::test_invalid_y_domain - AssertionError: 
FAILED tests/test_crs_transform_vectors.py::TestTransformVectors::test_invalid_xy_domain_corner - AssertionError: 
FAILED tests/test_polygon.py::TestMisc::test_infinite_loop_bounds[InterruptedGoodeHomolosine] - assert 3148724.315906706 == nan ± ???
FAILED tests/test_polygon.py::TestMisc::test_infinite_loop_bounds[Mollweide] - assert 0.0 == nan ± ???
FAILED tests/test_polygon.py::TestMisc::test_tiny_point_between_boundary_points - AssertionError: Got area 85832.84574015415, expecting ~81336
FAILED tests/test_vector_transform.py::Test_vector_scalar_to_grid::test_with_transform - AssertionError: 
= 39 failed, 625 passed, 7 skipped, 219 deselected, 2 xfailed, 7 warnings in 18.41s =</pre>
</details>
<details>
<summary>python314Packages.opensfm</summary>
<pre>
    def test_shot_view_ref_count() -&gt; None:
        &quot;&quot;&quot;Test that accessing shots via shot views maintains the map alive.&quot;&quot;&quot;
        rec = types.Reconstruction()
        camera1 = pygeometry.Camera.create_spherical()
        camera1.id = &quot;camera1&quot;
        rec.add_camera(camera1)
        rec.create_shot(&quot;shot1&quot;, &quot;camera1&quot;, pygeometry.Pose())
        rec.create_shot(&quot;shot2&quot;, &quot;camera1&quot;, pygeometry.Pose())
    
        # The reconstruction has ref count = 2
        count = sys.getrefcount(rec)
&gt;       assert count == 2
E       assert 1 == 2

opensfm/test/test_types.py:293: AssertionError
=========================== short test summary info ============================
FAILED opensfm/test/test_commands.py::test_run_all - _pickle.UnpicklingError: global &#x27;numpy._core.numeric._frombuffer&#x27; is forbidden
FAILED opensfm/test/test_types.py::test_shot_view_ref_count - assert 1 == 2
============ 2 failed, 250 passed, 1 deselected in 75.66s (0:01:15) ============</pre>
</details>
<details>
<summary>python314Packages.swh-scheduler</summary>
<pre>  +     &#x27;kwargs&#x27;: &#x27;[Filtered]&#x27;,
  -     &#x27;args&#x27;: [
  -         &#x27;arg&#x27;,
  -     ],
  -     &#x27;kwargs&#x27;: {
  -         &#x27;kwarg&#x27;: &#x27;kwarg&#x27;,
  -     },
        &#x27;task_name&#x27;: &#x27;swh.scheduler.tests.tasks.error&#x27;,
    }
====== 2 failed, 660 passed, 1 xfailed, 350 warnings in 167.34s (0:02:47) ======
stopping postgresql
waiting for server to shut down....2026-05-17 16:07:36.148 UTC [523] LOG:  received fast shutdown request
2026-05-17 16:07:36.148 UTC [523] LOG:  aborting any active transactions
2026-05-17 16:07:36.150 UTC [523] LOG:  background worker &quot;logical replication launcher&quot; (PID 529) exited with exit code 1
2026-05-17 16:07:36.151 UTC [524] LOG:  shutting down
2026-05-17 16:07:36.151 UTC [524] LOG:  checkpoint starting: shutdown immediate
2026-05-17 16:07:36.167 UTC [524] LOG:  checkpoint complete: wrote 975 buffers (6.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.015 s, sync=0.001 s, total=0.017 s; sync files=317, longest=0.001 s, average=0.000 s; distance=4507 kB, estimate=4507 kB; lsn=0/1953678, redo lsn=0/1953678
2026-05-17 16:07:36.182 UTC [523] LOG:  database system is shut down
 done
server stopped</pre>
</details>
</details>

--- a/pkgs/development/python-modules/mirakuru/default.nix
+++ b/pkgs/development/python-modules/mirakuru/default.nix
@@ -25,7 +25,7 @@ buildPythonPackage rec {
     hash = "sha256-3WyjvHxr+6kG+cLSCEZkHoA70mSoT66ubmp0W9g2yJM=";
   };
 
-  postPatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
+  postPatch = ''
     substituteInPlace tests/executors/test_output_executor_regression_issue_98.py \
       --replace-fail "timeout=15," "timeout=60,"
   '';

